tools/generate-baseline.js

```diff
@@ -33,17 +33,31 @@ const baselineIds = new Map([
 	[false, BASELINE_FALSE],
 ]);
 
+/**
+ * Encodes the baseline status and year fields into a single string.
+ * @param {{status: string, year: number}} baselineStatus The baseline fields to encode.
+ * @returns {string} The encoded baseline status and year.
+ */
+function encodeBaselineStatus({ status, year }) {
+	if (status === BASELINE_FALSE) {
+		// by definition, this status does not have a year, so omit it
+		return String(BASELINE_FALSE);
+	}
+
+	return `${status} (${year})`;
```

Comment:
I'm not sure how valuable the parentheses are here. Maybe for simpler parsing we could do this?
```suggestion
	return `${status}:${year}`;
```

Then we can just split on ":" and avoid regexs altogether.